.github/workflows/go.yml

```diff
@@ -32,6 +32,3 @@ jobs:
           STREAM_CHAT_API_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
           STREAM_CHAT_API_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
         run: go test -v -race ./...
-
-      - name: Build on ${{ matrix.goVer }}
-        run: go build ./...
```

Comment:
`go test` already builds a project so this build step was redundant.

Reply:
it might be useful sometimes if there is no test in the package, it will be ignored. In this case, it's flat, good call